epub33/core/index.html

```diff
@@ -4362,123 +4320,118 @@ <h4>Scripting Contexts</h4>
 							a future update adds the concept.</p>
```

Comment:
This is purely aesthetic: I would merge the two notes in one, and let the sentence on the data block follow the initial paragraph. The green boxes for notes break the flow...

Reply:
Merging doesn't really make sense in this case as the notes refer to two separate statements. It would jumble the discussion.

The first note could probably be regular spec prose, though.